features/borrower_requests_auth_code.feature

```diff
@@ -12,21 +12,3 @@ Scenario: Sign my mortgage page tells borrower they need their mobile phone
   Given I navigate to the borrower frontend "/sign-my-mortgage" page
   Then the text "the mobile phone you registered with your conveyancer" is displayed on the page
 
-@us46
-Scenario: 1 Borrower Requests Auth Code to sign deed
-  Given I create default deed with <1> borrowers
-  And borrower <1> views the deed
-  Then the deed is digitally signed by borrower <1>
-  And borrower <1> views the deed
-  And I verify borrower <1> has signed the deed
-
-@us46
-Scenario: 2 Borrowers Request Auth Code to sign deed
-  Given I create default deed with <2> borrowers
-  And borrower <1> views the deed
-  Then the deed is digitally signed by borrower <1>
-  And borrower <2> views the deed
-  And I verify borrower <1> has signed the deed
-  Then the deed is digitally signed by borrower <2>
-  And borrower <2> views the deed
-  And I verify borrower <2> has signed the deed
```

Comment:
I would want to keep one of these scenarios in as unless I've missed something I don't think this requesting the code bit is checked anywhere else?